Utility.cpp

```diff
@@ -1,108 +1,141 @@
 #include "Utility.h"
-
 #include "HelpfulItem.h"
 #include "DefensiveItem.h"
 #include "Character.h"
+#include "Dwarf.h"
+#include "Dragon.h"
+#include "DragonSlayer.h"
+#include "Paladin.h"
+
+#include <random>
 
+const int MIN_RANDOM_ITEM = 1;
+const int MAX_RANDOM_ITEM = 5;
 
 std::vector<std::unique_ptr<Item>> makeHelpfulItems(int num)
 {
     std::vector<std::unique_ptr<Item>> items;
-    
-    while( num-- >= 0 )
+
+    while (num-- > 0)
```

Comment:
Is this a bug fix?   Was the prior `while()` adding one extra element?  if so, nice catch.

Reply:
Yes, I noticed the random generator sometimes gives a character 6 items, but I'm pretty sure the range is set correctly.